File: geek-university/algoritmo-e-logica-de-programacao/python/exercicio_19.py
p = float(input("Informe o peso dos peixes: "))

if p > 50:
    e = (p - 50)
    m = e * 4.00
    print("Você deverá pagar R$ {0:.2f}".format(m))
else:
    m = 0
    e = 0
    print("Multas: {0}".format(m))
    print("Excesso: {0}".format(e))

File: geek-university/algoritmo-e-logica-de-programacao/java/exercicio_19.java
import java.util.Scanner;

public class exercicio_19 {

    public static void main(String[] args) {
        float p, m, e;
        Scanner teclado = new Scanner(System.in);

        System.out.print("Informe o peso dos peixes: ");
        p = teclado.nextFloat();

        if (p > 50) {
            e = (p - 50);
            m = (float) (e * 4.00);
            System.out.printf("Você devera pagar R$ %.2f em multas", m);
        } else {
            m = 0;
            e = 0;

            System.out.println("Multas: " + m);
            System.out.println("Excesso: " + e);
        }

        teclado.close();
    }

}
